README.md

```diff
@@ -0,0 +1,26 @@
+The task is to implement a "text translation pipeline": a backend and a web interface for a system in which translators take tasks from a list and translate the text specified in the task details. Translated tasks are then reviewed by Quality Assurance employees and are either sent to the client or returned back to the translator.
+
+There are 3 roles in this system:
+
+* **Administrators** can create and modify users and tasks (specifying the original text and, optionally, the date by which the task must be sent to the client)
+* **Translators** can take a task from the list of available tasks, provide a translation, then send it to quality control (with an optional comment), or return it back into "available" tasks
+* **Quality Assurance (QA)**  can take a task from the list of "translated" tasks, return it back to the (same) translator with a comment (in this case, when the translator subsequently submits the task for quality control again, the system will automatically send it to the same QA employee), return it back into the list of "translated" tasks, or send the results to the client
```

Comment:
All these Roles are not specified in an explicit form, which creates room for bugs and "unspecified possibilities".
Basically, any `User` with assigned permissions can perform related action, for example, `Translators` can `add qa comment` if they have related permission `translations.can_change_qa_comment`.
It means, that domain entities (from `Organization` domain) aren't specified properly in the codebase.